akvo/rsr/usecases/jobs/aggregation.py

```diff
@@ -42,18 +42,42 @@ def get_finished_jobs() -> QuerySet[IndicatorPeriodAggregationJob]:
 
 
 def base_get_jobs() -> QuerySet[IndicatorPeriodAggregationJob]:
-    return IndicatorPeriodAggregationJob.objects.select_related("period__indicator")
+    return IndicatorPeriodAggregationJob.objects.select_related("period__indicator").order_by("id")
 
 
-def schedule_aggregation_job(period: IndicatorPeriod) -> IndicatorPeriodAggregationJob:
+def schedule_aggregation_job(period: IndicatorPeriod) -> List[IndicatorPeriodAggregationJob]:
```

Comment:
Probably should be renamed to `schedule_aggregation_jobs`

Reply:
Fixed